server/rs_core/templates/registration/signup.html

```diff
@@ -4,19 +4,42 @@
 
 {% block content %}
   <div class="container">
-    <div class="row">
-        <div class="col-sm-12">
-            {% if user.is_anonymous %}
-                <h2 class="page-title">Sign Up</h2>
-                <div class="col-sm-6">
-                    <!-- signup -->
-                    {% include "registration/signup_form.html" %}
-                    <!-- signup -->
-                </div>
-            {% else %}
-                <p>You're already logged in. If you'd like to create a new account, you'll need to log out first.</p>
-            {% endif %}
+    <div class="col-xs-12 col-md-8 offset-md-2">
+      <h2>Register</h2>
+      
+      <p>
+        Complete the <em>Registration Form</em> below to... Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, voluptas!
+        Illo impedit incidunt totam dolor omnis quibusdam dolorem placeat rem quis eos.
+        Fugit, sequi ipsa. Id perspiciatis sit quia soluta odio excepturi,
+        impedit ipsa magni, mollitia, ab nobis quidem.
+      </p>
```

Comment:
One more place to remove placeholder text. Everything looks great other than this. I'll approve the PR after you remove this. @mbwatson